app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts

```diff
@@ -326,7 +327,8 @@ export class MapPreviewComponent implements OnInit, AfterViewInit, AfterViewChec
    * of the border of BC
    */
   get bcBorderCoordinates(): MapMarker[] {
-    const coordinates = this.bcGeoJSON.features[0].geometry.coordinates;
+    const coordinates = this.bcGeoJSON.features[0].geometry.coordinates[0][0];
+    console.dir(coordinates);
```

Comment:
do we still need to log these coords in the console?

Reply:
good catch